APKBUILD

```diff
@@ -25,8 +25,6 @@ package() {
   rm -rf "$pkgdir"/usr/glibc-compat/lib/gconv
   rm -rf "$pkgdir"/usr/glibc-compat/lib/getconf
   rm -rf "$pkgdir"/usr/glibc-compat/lib/audit
-  rm -rf "$pkgdir"/usr/glibc-compat/lib/*.a
-  rm -rf "$pkgdir"/usr/glibc-compat/include
```

Comment:
Here's the contents of the main `glibc` package, after this change. Note that no `lib/*.a` files or `include/` folder exists.
```
/packages # tar -xzf builder/x86_64/glibc-2.25-r1.apk
/packages # tree etc/ lib/ lib64/ usr/
etc/
├── ld.so.cache -> /usr/glibc-compat/etc/ld.so.cache
└── nsswitch.conf
lib/
└── ld-linux-x86-64.so.2 -> /usr/glibc-compat/lib/ld-linux-x86-64.so.2
lib64/
└── ld-linux-x86-64.so.2 -> /usr/glibc-compat/lib/ld-linux-x86-64.so.2
usr/
└── glibc-compat
    ├── etc
    │   ├── ld.so.cache
    │   └── ld.so.conf
    └── lib
        ├── ld-2.25.so
        ├── ld-linux-x86-64.so.2 -> ld-2.25.so
        ├── libBrokenLocale-2.25.so
        ├── libBrokenLocale.so -> libBrokenLocale.so.1
        ├── libBrokenLocale.so.1 -> libBrokenLocale-2.25.so
        ├── libSegFault.so
        ├── libanl-2.25.so
        ├── libanl.so -> libanl.so.1
        ├── libanl.so.1 -> libanl-2.25.so
        ├── libc-2.25.so
        ├── libc.so
        ├── libc.so.6 -> libc-2.25.so
        ├── libcidn-2.25.so
        ├── libcidn.so -> libcidn.so.1
        ├── libcidn.so.1 -> libcidn-2.25.so
        ├── libcrypt-2.25.so
        ├── libcrypt.so -> libcrypt.so.1
        ├── libcrypt.so.1 -> libcrypt-2.25.so
        ├── libdl-2.25.so
        ├── libdl.so -> libdl.so.2
        ├── libdl.so.2 -> libdl-2.25.so
        ├── libm-2.25.so
        ├── libm.so
        ├── libm.so.6 -> libm-2.25.so
        ├── libmemusage.so
        ├── libmvec-2.25.so
        ├── libmvec.so -> libmvec.so.1
        ├── libmvec.so.1 -> libmvec-2.25.so
        ├── libnsl-2.25.so
        ├── libnsl.so -> libnsl.so.1
        ├── libnsl.so.1 -> libnsl-2.25.so
        ├── libnss_compat-2.25.so
        ├── libnss_compat.so -> libnss_compat.so.2
        ├── libnss_compat.so.2 -> libnss_compat-2.25.so
        ├── libnss_db-2.25.so
        ├── libnss_db.so -> libnss_db.so.2
        ├── libnss_db.so.2 -> libnss_db-2.25.so
        ├── libnss_dns-2.25.so
        ├── libnss_dns.so -> libnss_dns.so.2
        ├── libnss_dns.so.2 -> libnss_dns-2.25.so
        ├── libnss_files-2.25.so
        ├── libnss_files.so -> libnss_files.so.2
        ├── libnss_files.so.2 -> libnss_files-2.25.so
        ├── libnss_hesiod-2.25.so
        ├── libnss_hesiod.so -> libnss_hesiod.so.2
        ├── libnss_hesiod.so.2 -> libnss_hesiod-2.25.so
        ├── libnss_nis-2.25.so
        ├── libnss_nis.so -> libnss_nis.so.2
        ├── libnss_nis.so.2 -> libnss_nis-2.25.so
        ├── libnss_nisplus-2.25.so
        ├── libnss_nisplus.so -> libnss_nisplus.so.2
        ├── libnss_nisplus.so.2 -> libnss_nisplus-2.25.so
        ├── libpcprofile.so
        ├── libpthread-2.25.so
        ├── libpthread.so
        ├── libpthread.so.0 -> libpthread-2.25.so
        ├── libresolv-2.25.so
        ├── libresolv.so -> libresolv.so.2
        ├── libresolv.so.2 -> libresolv-2.25.so
        ├── librt-2.25.so
        ├── librt.so -> librt.so.1
        ├── librt.so.1 -> librt-2.25.so
        ├── libthread_db-1.0.so
        ├── libthread_db.so -> libthread_db.so.1
        ├── libthread_db.so.1 -> libthread_db-1.0.so
        ├── libutil-2.25.so
        ├── libutil.so -> libutil.so.1
        ├── libutil.so.1 -> libutil-2.25.so
        └── locale

4 directories, 74 files
```